conversation_message.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"accounts-service/models"
+	accountsv1 "accounts-service/protorepo/noted/accounts/v1"
+	"accounts-service/validators"
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func (server *conversationsAPI) SendConversationMessage(ctx context.Context, in *accountsv1.SendConversationMessageRequest) (*accountsv1.SendConversationMessageResponse, error) {
+	err := validators.ValidateSendConversationMessageRequest(in)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	token, err := server.authenticate(ctx)
+	if err != nil {
+		return nil, status.Error(codes.Unauthenticated, err.Error())
+	}
+
+	conversation, err := server.repo.Get(ctx, &models.OneConversationFilter{ID: in.ConversationId})
+	if err != nil {
+		return nil, statusFromModelError(err)
+	}
+
+	accountId := token.UserID.String()
+
+	_, err = server.groupService.GetGroupMember(ctx, &accountsv1.GetGroupMemberRequest{GroupId: conversation.GroupID, AccountId: accountId})
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, "not in group")
```

Comment:
dk but i think we should return directly the err from GetGroupMember (if it's internal or not found it will handle itself on it's own)